app/routes.py

```diff
@@ -240,16 +240,20 @@ def upgrade():
 @core.route('/environments')
 def environments():
     webhook = current_app.ldclient.variation('environments-webhook', current_user.get_ld_user(), False)
-
-    if webhook:
-        try:
-            ld = LaunchDarklyApi(os.environ.get('LD_API_KEY'))
-            project = ld.get_project(PROJECT_NAME)
-            project_pick = pickle.dumps(project)
-            current_app.redis_client.set(PROJECT_NAME, project_pick)
-            return jsonify({'response': 200})
-        except Exception as e:
-            current_app.logger.error(e)
-            return jsonify({'response': 400})
+    url = url_parse(request.url)
+    subdomain = url.host.split('.')[0]
+    if subdomain == 'admin':
+        if webhook:
```

Comment:
@BrianJayChung can we merge these 2 if statements together using `and`?

Reply:
If they are merged how should we track between domain not matching and flag evaluated, since they will both hit abort(404)

Reply:
do we generally need to discern between the differences?

Reply:
Not really, since this is really only meant to be accessed by a few.